src/charm.py

```diff
@@ -226,6 +225,7 @@ def _on_database_changed(self, event: DatabaseEndpointsChangedEvent) -> None:
 
     def _on_config_changed(self, event: ConfigChangedEvent) -> None:
         self.config_file.base_dn = self.config.get("base_dn")
+        self.config_file.starttls_config = StartTLSConfig.load(self)
```

Comment:
Isn't this a duplicate of https://github.com/canonical/glauth-k8s-operator/pull/22/files#diff-b9ed39bbc9c0387bd3e07da31d13373745534a1cd723d3e292c73496b12e307cR146?

We load the `startTLSConfig` on `__init__`, no need to do it again here (unless I'm missing something)